azure-pipelines.yml

```diff
@@ -0,0 +1,70 @@
+trigger:
+  batch: true
+  branches:
+    include:
+      - "*"
+
+pr: none
+
+workspace:
+  clean: all
+  
+variables:
+- name: BuildPlatform
+  value : 'any cpu'
+- name: BuildConfiguration
+  value: 'release'
+- group: 'Prod Management Resources'
+
+resources:
+  repositories:
+  - repository: das-platform-building-blocks
+    type: github
+    name: SkillsFundingAgency/das-platform-building-blocks
+    ref: refs/heads/master
+    endpoint: 'GitHub (SFA)'
+
+pool:
+  name: 'DAS - Continuous Integration'
+
+steps:
+  - task: DotNetCoreCLI@2
```

Comment:
You can reference the step template we have in platform-building-blocks 

  - template: azure-pipelines-templates/build/step/app-build.yml@das-platform-building-block